titus-common/src/main/java/com/netflix/titus/common/util/FunctionExt.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.common.util;
+
+import java.util.function.Predicate;
+
+public final class FunctionExt {
+    public static <T> Predicate<T> alwaysTrue() {
+        return ignored -> true;
```

Comment:
Make it a constant to avoid object construction on each invocation.

Reply:
can't because of generic types. Unless I force the cast?

Reply:
Casting should be ok here.

Reply:
OK, will do.